api/policy.go

```diff
@@ -15,11 +17,17 @@ type PipelineApprovalValue string
 // BackupPlanPolicySchedule is value for backup plan policy.
 type BackupPlanPolicySchedule string
 
+// AllowedWindowPeriodCron is value for allowed window period policy, and is stored in cron format.
+// a valid cron expression should be parsed without any error by this: cron.NewParser(cron.Minute | cron.Hour | cron.Dom | cron.Month | cron.Dow)
+type AllowedWindowPeriodCron string
```

Comment:
Need to elaborate a bit on how to design this, let's do a design doc

Reply:
Please refer to [https://bytebase.feishu.cn/docs/doccnImYPzC7dWQF7VxdMEcMDyb](url)